Tests/UIKitTests/UIViewControllerExtensionsTests.swift

```diff
@@ -22,6 +22,10 @@ final class UIViewControllerExtensionsTests: XCTestCase {
         }
     }
 
+    class MyViewController: UIViewController {
```

Comment:
You need to move this class declaration outside of the test class otherwise the storyboard won't recognise it. In fact maybe just make a new file for it in the `ResourcesTests` folder.